### PR TITLE
DEV-180: Lilypad Launcher: (Mac) Launcher progress indicator

### PR DESCRIPTION
**Pull request by danteruiz**
Ticket - https://highfidelity.atlassian.net/secure/RapidBoard.jspa?rapidView=46&modal=detail&selectedIssue=DEV-180&assignee=557058%3A56835251-5116-4703-a5d3-30bccdb21af4

also addresses this ticket https://highfidelity.atlassian.net/browse/BUGZ-873

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20279/HighFidelity-Beta-PR15900-15cc7a0.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20279/HighFidelity-Beta-Interface-PR15900-15cc7a0.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20279/HighFidelity-Beta-PR15900-15cc7a0.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20279/HighFidelity-Quest-Beta-PR15900-15cc7a0.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20292/HighFidelity-Beta-PR15900-1741b9a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20292/HighFidelity-Beta-Interface-PR15900-1741b9a.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20292/HighFidelity-Beta-PR15900-1741b9a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20292/HighFidelity-Beta-Interface-PR15900-1741b9a.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20292/HighFidelity-Beta-PR15900-1741b9a.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20292/HighFidelity-Quest-Beta-PR15900-1741b9a.apk)

File: launchers/darwin/src/Launcher.m [hunk 1]
```diff
@@ -207,8 +274,12 @@ - (void) displayNameEntered:(NSString*)aDiplayName
...
 - (void) domainContentDownloadFinished
 {
-    //.[self.downloadScripts downloadScripts:self.domainScriptsUrl];
-    [self.latestBuildRequest requestLatestBuildInfo];
+    if (self.shouldDownloadInterface) {
+        [self.downloadInterface downloadInterface: self.interfaceDownloadUrl];
+        return;
+    }
+    [self interfaceFinishedDownloading];
+    //[self.latestBuildRequest requestLatestBuildInfo];
```
Comment (luiscuenca) [inline launchers/darwin/src/Launcher.m]: Should we keep this comment?

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20296/HighFidelity-Beta-PR15900-996a2f0.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20296/HighFidelity-Quest-Beta-PR15900-996a2f0.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20296/HighFidelity-Beta-PR15900-996a2f0.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20296/HighFidelity-Beta-Interface-PR15900-996a2f0.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20296/HighFidelity-Beta-PR15900-996a2f0.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20296/HighFidelity-Beta-Interface-PR15900-996a2f0.exe

**Pull request merged**